functions/__sf_section_dir.fish

```diff
@@ -35,9 +40,13 @@ function __sf_section_dir -d "Display the current truncated directory"
 	# Truncate the path to have a limited number of dirs
 	set dir (__sf_util_truncate_dir $tmp $SPACEFISH_DIR_TRUNC)
 
+    if [ $SPACEFISH_DIR_LOCK_SHOW = true -a ! -w . ];
+        set DIR_LOCK_SYMBOL (set_color $SPACEFISH_DIR_LOCK_COLOR)" $SPACEFISH_DIR_LOCK_SYMBOL"(set_color --bold fff)
```

Comment:
Indentation is borked here. If/end statements are having a dance!

Reply:
(Not visible from GitHub's preview, you'd have to view the whole file to see the `end` statement.)